reporting/reports/data/system-history

```diff
@@ -52,6 +54,8 @@ sql:
 				end as event_data
 		from rhnserveraction left outer join rhnactionerrataupdate
 			on rhnserveraction.action_id = rhnactionerrataupdate.action_id
+			left outer join rhnserver 
+				on rhnserveraction.server_id = rhnserver.id
```

Comment:
As `id` is the primary key in `rhnServer` and `server_id` can't be `NULL` in `rhnServerAction`, I'd suggest an inner join here.

https://github.com/uyuni-project/uyuni/blob/master/schema/spacewalk/common/tables/rhnServerAction.sql#L22

Reply:
Thank you @moio. I have changed the join to inner join.